test/integration_test.go

```diff
@@ -6976,6 +6976,25 @@ func (ts *IntegrationTestSuite) TestUserMetadata() {
 	ts.Equal("my-child-wf-details", str)
 }
 
+func (ts *IntegrationTestSuite) TestTaskQueuePriority() {
```

Comment:
Would be neat to have a simple behavior test to confirm it works, but a test confirming server has received it is good enough